docs/scos/dev/back-end-development/project-namespaces.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Project namespaces
+description:
+last_updated: Nov 22, 2022
+template: howto-guide-template
+---
+
+In Spryker, the project namespace is used to identify and distinguish your project from other projects within the framework. It helps to ensure that the code and resources specific to your project are properly organized and separated from the rest of the framework.
+
+Using a project namespace also helps to prevent naming conflicts with other projects, as it allows you to use unique names for your project's classes, functions, and other code elements. This can be especially important when working with a large, complex framework like Spryker, as it can help to keep your code organized and maintainable.
+Overall, the project namespace is an important part of Spryker framework and can help to improve the reliability, scalability, and maintainability of your project.
+
+At Spryker the default project name is Pyz and the project code is located in: `src/Pyz` folder. According to the PSR-4 standard PHP namespace has to correlate with folder name. In this case the namespace must be “Pyz” and followed that in a subfolder e.g. `src/Pyz/Zed/Cart` the namespace will be `Pyz\Zed\Cart`.
+
+Projects can be renamed but to follow up the PSR-4 standard namespaces must be rewritten in all php files in projects directory and all sub-directories.
+In the case of creating new directories for projects source codes it’s also mandatory to follow the namespace convention, started the naming from the src directory level.
+
+## How to configure project namespaces
+
+After You decided the unique custom namespace of Your project e.g. "*MyProject*", you will need to follow these steps:
+
+1, Create a directory for your namespace under the src directory of your Spryker project. The directory should have the same name as your namespace, with each part of the namespace separated by a directory separator (e.g., `src/MyProject` for the *MyProject* namespace).
```

Comment:
not sure if it syntax, but I guess we should use dot instead of comas in the list